### ISSUE TITLE
Proxy not working anymore

### ISSUE THREAD

**Issue by crystalin** [polkadot-js/apps#8463]
Environment:
* Moonbeam (AccountId20) production network
* Injected Talisman wallet

The scenario is:
* I load my keys into talisman wallet and connect to the dapp
![image](https://user-images.githubusercontent.com/329248/205238129-33f874d2-ad9d-41e1-b666-c1e083c6bebc.png)
* I load the proxied account into the dapp (only the address)
![image](https://user-images.githubusercontent.com/329248/205238035-d1d21d94-3b87-421b-b0e4-ebf92991b8cb.png)
* I try to perform an action from the proxy account (only the proxied account is suggested)
![image](https://user-images.githubusercontent.com/329248/205238230-5c53cf2f-1331-4411-be98-7c4679229bbb.png)

* I get error 
![image](https://user-images.githubusercontent.com/329248/205238333-caa02900-2478-4f14-8e4f-072ed5236880.png)

**Comment by jacogr** [comment]
There is a manual mapping from allowed pallets to proxy types. Since the above is not a known location, it is not available.

**Comment by crystalin** [comment]
Thank you,
Can it be override for specific chain? If yes please let me know and I'll add it to our list of items to maintain :)

**Comment by jacogr** [comment]
No, it is not on a per-chain/spec basis. However, for most of these it should not have conflicts, i.e. `council` (currently there) and `councileCollective` (not there) _should_ really not have issues between chains.

**Comment by crystalin** [comment]
I see,
I'm mostly concerned about Governance, nonTransfer which are differents
